test/test_suite_runner.py

```diff
@@ -5,15 +5,15 @@
 
 class TestSuiteRunner:
     test_cloud_provider = 'aws'
```

Comment:
Shouldn't we get this from resources.json? I know it's just for debugging purposes, but I think we could implement it now